src/app/Home/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,7 @@
+
+<div class="sidebar">
+    <div class="sidebar-section" (click)="navigateTo('showBooks')">Books</div>
+    <div class="sidebar-section" (click)="navigateTo('genres')">Genres</div>
+    <div class="sidebar-section" (click)="navigateTo('authors')">Authors</div>
+    <div class="sidebar-section" (click)="navigateTo('issued-books')">Issued Books</div>
```

Comment:
The use of a <div> element in this context is semantically inappropriate.

Reply:
What can i use instead of dive apu? do you have any suggestion?
I've changed it to routerLink now. thank you